test/old/testdir/test_filetype.vim

```diff
@@ -348,6 +348,7 @@ func s:GetFilenameChecks() abort
     \ 'ipfilter': ['ipf.conf', 'ipf6.conf', 'ipf.rules'],
     \ 'iss': ['file.iss'],
     \ 'ist': ['file.ist', 'file.mst'],
+    \ 'inko': ['file.inko'],
```

Comment:
This should be between `inittab` and `ipfilter`

Reply:
@zeertzjq The list itself is fairly arbitrary (i.e. it's only partially alphabetical, the rest is seemingly a mess). Is the need for this documented somewhere?